### PR TITLE
sacc: 1.01 → 1.02

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Motivation for this change

Update `sacc` to 1.02.

cc @rapenne-s

Also switched download from `fetchgit` to `fetchurl` using the gopher protocol. Are there any objections to this? Fetching the tarball via gopher is faster than checking out the git repository so I think it's desireable, the question is if we want to support this protocol indefinitely (I guess if we continue using curl we will).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
@GrahamcOfBorg build sacc

**Comment by sternenseemann** [comment]
Duplicate (with the difference of the switch to gopher) of #98850

**Comment by rapenne-s** [comment]
I can compile it and it works.
Tested on NixOS amd64.

**Review by rapenne-s — APPROVED** [review]
LGTM, build and works on amd64

**Comment by NinjaTrappeur** [comment]
Result of `nixpkgs-review pr 98852` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sacc</li>
  </ul>
</details>

**Comment by NinjaTrappeur** [comment]
Thanks!

I'm still not sure how I feel about the gopher fetchurl. 

Curl is unlikely to drop the gopher support anytime soon, however we currently do not fetch anything  via this protocol on nixpkgs.

I'm gonna wait for a second opinion before merging that.

**Review by picnoir — APPROVED** [review]

**Comment by JJJollyjim** [comment]
looks like it's also available at ftp://bitreich.org/releases/sacc/sacc-1.02.tgz

**Comment by sternenseemann** [comment]
Good catch, FTP is definitely a better option for downloading binary files than Gopher. Force pushed to use ftp.

**Comment by NinjaTrappeur** [comment]
Nit: it's best to stick to `->` in the commit message instead of using the `→` symbol.

https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

**Pull request merged**
